### PR TITLE
Sections 2

### PR DESCRIPTION
**Pull request by SergioMiguez**

### CODE REVIEW

File: layouts/partials/banner.html [hunk 1]
```diff
@@ -3,25 +3,37 @@
 {{ if $data.banner.banner.enable }}
 {{ with $data.banner.banner }}
 {{"<!-- Welcome Slider-->" | safeHTML}}
-<section class="hero-area" style='background-image: url("{{ .bg_image_webp | absURL }}")'>
+<section class="hero-area">
+<!--<section class="hero-area" style='background-image: url("{{ .bg_image_webp | absURL }}")'>-->
```
Comment (nankeen) [inline layouts/partials/banner.html]:
```suggestion
```

Oops I forgot, we should remove the comments.

File: layouts/partials/banner.html [hunk 2]
```diff
@@ -3,25 +3,37 @@
...
 	<!-- this image is for trigger this section fallback background image -->
```
Comment (nankeen) [inline layouts/partials/banner.html]:
```suggestion
```

Here too.

File: layouts/partials/banner.html [hunk 3]
```diff
@@ -3,25 +3,37 @@
...
-	<img class="d-none" src="{{ .bg_image_webp | absURL }}" onerror="this.onerror=null;this.parentElement.style.backgroundImage= 'url({{ .bg_image | absURL }})'"/>
+	<!--<img class="d-none" src="{{ .bg_image_webp | absURL }}" onerror="this.onerror=null;this.parentElement.style.backgroundImage= 'url({{ .bg_image | absURL }})'"/>-->
```
Comment (nankeen) [inline layouts/partials/banner.html]:
```suggestion
```

**Review by nankeen — COMMENTED** [review]
Forgot about the HTML comments, we probably won't need them now

**Review by nankeen — APPROVED** [review]
LGTM. Let's merge.

**Comment by SergioMiguez** [comment]
Perfect

**Pull request merged**